### ISSUE TITLE
Access to datasets not preprocessed

### ISSUE THREAD

**Issue by urospet** [rashadulrakib/short-text-clustering-enhancement#6]
Is it possible to have access to the datasets not preprocessed, especially tweets and google news. I was not able to find them.

**Comment by rashadulrakib** [comment]
Hello,

I think the datasets are not preprocessed in my github link

https://raw.githubusercontent.com/rashadulrakib/short-text-clustering-enhancement/master/data/agnewsdataraw-8000

https://raw.githubusercontent.com/rashadulrakib/short-text-clustering-enhancement/master/data/biomedical/biomedical_true_text

https://raw.githubusercontent.com/rashadulrakib/short-text-clustering-enhancement/master/data/stackoverflow/stackoverflow_true_text

**Comment by urospet** [comment]
You are right, about these three datasets! Thanks! How about Google News, it seems already preprocessed
